netlify.toml

```
@@ -4,3 +4,8 @@
 [build]
   publish = "docs/.vitepress/dist"
   command = "npx pnpm i --store=node_modules/.pnpm-store && npm run docs:build:ci"
+[[redirects]]
+from = "https://chat.fakerjs.dev"
+to = "https://discord.com/invite/4qDjAmDj4P"
+status = 301
+force = true
```

Is the formatting here correct?
The section above looks slightly different.

---

@ST-DDT This is what I used to write this formatting, it looks a bit different than the sections above.

https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

---

I will approve this once it uses consistent formatting. The example uses indented formatting, so we should go for that
```suggestion
[[redirects]]
  from = "https://chat.fakerjs.dev"
  to = "https://discord.com/invite/4qDjAmDj4P"
  status = 301
  force = true
```

---

> I will approve this once it uses consistent formatting. The example uses indented formatting, so we should go for that
> 
> ```suggestion
> 
> [[redirects]]
> 
>   from = "https://chat.fakerjs.dev"
> 
>   to = "https://discord.com/invite/4qDjAmDj4P"
> 
>   status = 301
> 
>   force = true
> 
> ```

Oh, I thought you were referencing the double brackets around "redirects". Committed